lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -391,51 +398,54 @@ define((require, exports, module) ->
       }
 
 
-  exports.ContextHandler = ContextHandler
-  exports.ConstrainedTokenIterator = ConstrainedTokenIterator
-  exports.EquationRangeHandler = EquationRangeHandler
-  exports.setupPreviewer = (editor, popoverHandler, katexLoader, I18N) ->
-    myKatexLoader = katexLoader
+  myContextHandler = null
+  reset = -> if myContextHandler? and myContextHandler.contextPreviewExists then myContextHandler.destroyEverything()
```

Comment:
I don't really like this because that means, that if `myContextHandler` does not exist, then, `myContextHandler?.contextPreviewExists` returns `undefined`, and then, because of the `if` condition, `undefined` is casted to `false`. And I really don't like, when anything is casted in JS. But okay, I'll do it.